data.ts

```diff
@@ -12,6 +12,7 @@ import { pathToFileURL } from 'url'
 import { Client } from '@opensearch-project/opensearch'
 import type { ClientOptions } from '@opensearch-project/opensearch'
 import { exists } from './paths'
+import _ from 'lodash'
```

Comment:
```suggestion
import chunk from 'lodash/chunk'
```

See the section titled "But lodash isn't lightweight enough! " at https://lodash.com/per-method-packages.

Reply:
Got it. Fixed.